bin/setup

```diff
@@ -24,8 +24,7 @@ FileUtils.chdir APP_ROOT do
   FileUtils.cp ".env.sample", ".env" unless File.exist?(".env")
 
   puts "\n== Preparing database =="
-  system! "bin/rails db:prepare"
-  system! "bin/rails db:sample_data"
+  system! "bin/rails db:prepare db:sample_data"
```

Comment:
Micro-optimization: make these two tasks a single `bin/rails` call to save on boot time.